pkg/resource/webhook_certificates.go

```diff
@@ -29,13 +29,14 @@ import (
 )
 
 const (
-	mutatingHookConfig   = "mutating-webhook-configuration"
+	// TODO: don't rely on config names
```

Comment:
This needs to be addressed. Because we don't control the namespace the `cockroach-operator-` prefix shouldn't be assumed. 

I'm thinking we could list the webhooks and patch all configs named `mcrdbcluster.kb.io` and `vcrdbcluster.kb.io`. Thoughts?

Reply:
That seems reasonable to me. Are there similar concerns about `webhookService`?

Reply:
Yes, essentially anything prefixed here with `cockroach-operator-`

Reply:
After discussing offline I think addressing this TODO is relatively low-priority; I'm fine moving forward